content/attacks/posts/2018-11-AurumCoin-Cryptopia

```diff
@@ -0,0 +1,31 @@
+---
+date: 2018-11
+custodians: AurumCoin, Cryptopia
+categories: 51%
+title: AurumCoin Users Face Losses in a 51% Attack
+---
+
+## Summary
+
+In November 2018, AurumCoin (AU), a relatively lesser-known cryptocurrency, was hit by a 51% attack. During the attack, an anonymous entity was able to gain control of a majority of the network's hash rate, which enabled them to alter the blockchain ledger and perform double-spend transactions. The attack resulted in the loss of 15,752.26 AurumCoins from the Cryptopia exchange, which was valued at approximately $500,000 at the time.
+
+## Attackers
+
+The attackers behind the 51% attack on AurumCoin remain unidentified. They were able to accumulate a majority of the network’s hash power, thereby taking control of the AurumCoin blockchain. This control enabled them to alter the blockchain ledger to their advantage and execute double-spend transactions.
```

Comment:
dont simply restate info from the summary. add info about the attackers or just leave the comment that they are unknown.